### PR TITLE
feat: unserve v1alpha1 version of platformadmin crd

### LINKED ISSUE #1656: [feature request] Unserve version alpha1 of platformadmin crd
**What would you like to be added**:
Remove version alpha1 of PlatformAdmin

**Why is this needed**:
Users are recommended to use the more full-featured version of v1alpha2.

**others**
/kind feature

### PR DESCRIPTION
**Pull request by LavenderQAQ**
#### What type of PR is this?
/kind enhancement
/sig iot

#### What this PR does / why we need it:
Let users use the more complete version of v1alpha2.

#### Which issue(s) this PR fixes:

Fixes #1656

#### Does this PR introduce a user-facing change?
```release-note
Please use the v1alpha2 version of platformadmin, which is much more complete than v1alpha1.
```

### CODE REVIEW

File: charts/yurt-manager/crds/iot.openyurt.io_platformadmins.yaml [hunk 1]
```diff
@@ -18,8372 +18,6 @@ spec:
...
-                                            type: object
-                                          failureThreshold:
-                                            description: Minimum consecutive failures
-                                              for the probe to be considered failed
-                                              after having succeeded. Defaults to
-                                              3. Minimum value is 1.
-                                            format: int32
-                                            type: integer
-                                          httpGet:
-                                            description: H
```
Comment (zyjhtangtang) [inline charts/yurt-manager/crds/iot.openyurt.io_platformadmins.yaml:8384]: It is recommended to keep the v1alpha1 version and set served to false

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**